app/src/main/res/values-el-rGR/strings.xml

```diff
@@ -282,7 +282,7 @@
     <string name="multipleBarcodesFoundPleaseChooseOne">Ποιους από τους γραμμωτούς κώδικες που βρέθηκαν θέλετε να χρησιμοποιήσετε;</string>
     <string name="pageWithNumber">Σελίδα <xliff:g>%d</xliff:g></string>
     <string name="spend">Δαπάνησε</string>
-    <string name="receive">Λαμβάνω</string>
```

Comment:
@stavpup Translators say this translates to "Download" instead of "Receive", which doesn't seem right.

Reply:
Receive = λαμβάνω
Download = κατεβάζω

Reply:
Oh huh, maybe I copied the wrong one, sorry :sweat_smile: